src/ports.js

```diff
@@ -0,0 +1,15 @@
+function Port (name) {
```

Comment:
As with the ships file, the name of the file and the class is different. As with that file, why not use a constructor to create a class?